src/app/pages/BridgeDepositPage/saga.ts

```diff
@@ -0,0 +1,346 @@
+import {
+  call,
+  cancel,
+  cancelled,
+  fork,
+  put,
+  select,
+  take,
+  takeLatest,
+} from 'redux-saga/effects';
+import delay from '@redux-saga/delay-p';
+import { actions } from './slice';
+import { actions as walletProviderActions } from 'app/containers/WalletProvider/slice';
+import {
+  getBridgeChain,
+  getBridgeChainId,
+  getSupportedBridgeChainIds,
+} from './utils/helpers';
+import { walletService } from '@sovryn/react-wallet';
+import { debug } from '@sovryn/common';
+import { eventChannel } from 'redux-saga';
+import { BridgeDepositPageState, DepositStep, TxStep } from './types';
+import { Chain } from '../../../types';
+import { selectBridgeDepositPage } from './selectors';
+import { BridgeDictionary } from './dictionaries/bridge-dictionary';
+import { BridgeModel } from './types/bridge-model';
+import { AssetModel } from './types/asset-model';
+import { bridgeNetwork } from './utils/bridge-network';
+import { bignumber } from 'mathjs';
+import { ethers } from 'ethers';
+
+const { log } = debug('bridge/saga.ts');
```

Comment:
I would delete logging, it shouldn't be in a production code.

Reply:
Logs will be silent unless user enables it on Localstorage

Reply:
ok then